cmd/podman/trust.go

```diff
@@ -0,0 +1,557 @@
+			Usage: "Trust type, accept values: signedBy(default), accept, reject.",
+		},
+		cli.StringSliceFlag{
+			Name: "pubkeysfile, f",
+			Usage: `Path of installed public key(s) to trust for TARGET.
+	Absolute path to keys is added to policy.json. May
+	used multiple times to define multiple public keys.
+	File(s) must exist before using this command.`,
+		},
+		cli.StringFlag{
+			Name:   "policypath",
+			Hidden: true,
+		},
+	}
+	showTrustFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "raw",
+			Usage: "Output raw policy file",
+		},
+		cli.BoolFlag{
+			Name:  "json, j",
+			Usage: "Output as json",
+		},
+		cli.StringFlag{
+			Name:   "policypath",
+			Hidden: true,
+		},
+		cli.StringFlag{
+			Name:   "registrypath",
+			Hidden: true,
+		},
+	}
+
+	setTrustDescription = "Set default trust policy or add a new trust policy for a registry"
+	setTrustCommand     = cli.Command{
+		Name:         "set",
+		Usage:        "Set default trust policy or a new trust policy for a registry",
+		Description:  setTrustDescription,
+		Flags:        sortFlags(setTrustFlags),
+		ArgsUsage:    "default | REGISTRY[/REPOSITORY]",
+		Action:       setTrustCmd,
+		OnUsageError: usageErrorHandler,
+	}
+
+	showTrustDescription = "Display trust policy for the system"
+	showTrustCommand     = cli.Command{
+		Name:                   "show",
+		Usage:                  "Display trust policy for the system",
+		Description:            showTrustDescription,
+		Flags:                  sortFlags(showTrustFlags),
+		Action:                 showTrustCmd,
+		ArgsUsage:              "",
+		UseShortOptionHandling: true,
+		OnUsageError:           usageErrorHandler,
+	}
+
+	trustSubCommands = []cli.Command{
+		setTrustCommand,
+		showTrustCommand,
+	}
+
+	trustDescription = "Manages the trust policy of the host system. Trust policy describes a registry scope that must be signed by public keys."
+	trustCommand     = cli.Command{
+		Name:         "trust",
+		Usage:        "Manage system container trust policy",
+		Description:  trustDescription,
+		ArgsUsage:    "{set,show} ...",
+		Subcommands:  trustSubCommands,
+		OnUsageError: usageErrorHandler,
+	}
+)
+
+func showTrustCmd(c *cli.Context) error {
+	runtime, err := libpodruntime.GetRuntime(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not create runtime")
+	}
+
+	var (
+		policyPath              string
+		systemRegistriesDirPath string
+	)
+	if c.IsSet("policypath") {
+		policyPath = c.String("policypath")
+	} else {
+		policyPath = defaultPolicyPath(runtime.SystemContext())
+	}
+	policyContent, err := ioutil.ReadFile(policyPath)
+	if err != nil {
+		return errors.Wrapf(err, "unable to read %s", policyPath)
+	}
+	if c.IsSet("registrypath") {
+		systemRegistriesDirPath = c.String("registrypath")
+	} else {
+		systemRegistriesDirPath = registriesDirPath(runtime.SystemContext())
+	}
+	if c.Bool("raw") {
+		_, err := os.Stdout.Write(policyContent)
+		if err != nil {
+			return errors.Errorf("could not read trust policies")
```

Comment:
You have eaten the error.

return errors.Wrap(err, "could not read trust policies")